From 57877e5b5cf2364557202fa27cad5c41f5d64bf1 Mon Sep 17 00:00:00 2001
From: Charles Keepax <ckeepax@opensource.wolfsonmicro.com>
Date: Thu, 30 Jan 2014 18:42:06 +0000
Subject: [PATCH 1/2] ASoC: arizona: An OUTDIV of 1 is not valid, avoid this

One is not a valid value for the OUTDIV start searching at 2 instead.

Change-Id: Ibeda365afbe7f25778da7f3c185e474a5565200c
Signed-off-by: Charles Keepax <ckeepax@opensource.wolfsonmicro.com>
---
 sound/soc/codecs/arizona.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/codecs/arizona.c b/sound/soc/codecs/arizona.c
index f5e9285..ba575a9 100644
--- a/sound/soc/codecs/arizona.c
+++ b/sound/soc/codecs/arizona.c
@@ -1268,7 +1268,7 @@ static int arizona_calc_fll(struct arizona_fll *fll,
 	Fref /= div;
 
 	/* Fvco should be over the targt; don't check the upper bound */
-	div = 1;
+	div = 2;
 	while (Fout * div < 90000000 * fll->vco_mult) {
 		div++;
 		if (div > 7) {
-- 
1.7.10.4

